receiver/splunkenterprisereceiver/README.md

```diff
@@ -8,7 +8,7 @@ jobs.
 
 ## Configuration
 
-The following settings are required, omitting them will either cause your receiver to fail to compile or result in 4/5xx return codes during scraping.
+The following settings are required, omitting them will either cause your receiver to fail to compile or result in 4/5xx return codes during scraping. These must be set for each Splunk instance type (indexer, search head, or cluster master) from which you wish to pull metrics. At present, only one of each type is accepted, per configured receiver instance.
```

Comment:
Just to clarify here. `only one of each type is accepted` means that if I had 3 indexers I'd need to have 3 different `splunkenterprise` receiver entries each with 1 of the indexers included.
Is my reading of that right?

This isn't an issue mostly just a clarification question. TY!